tests/mongo-serde/build.gradle

```diff
@@ -3,7 +3,16 @@ plugins {
     id "io.micronaut.build.internal.mongodb-functional-test"
 }
 
+micronaut {
+    importMicronautPlatform = false
+}
+
 dependencies {
+    micronautBoms(
```

Comment:
Hacky workaround until platform gets serde M13. There is probably a cleaner way but I am not so familiar with gradle

Reply:
I think this fix is fine.